templates/default/kitchen.cloud.yml.erb

```diff
@@ -1,14 +1,17 @@
 ---
 driver_config:
-  openstack_username: <\%= ENV['OS_USERNAME'] %>
-  openstack_api_key: <\%= ENV['OS_PASSWORD'] %>
-  openstack_auth_url: <\%= "#{ENV['OS_AUTH_URL']}/tokens" %>
-  key_name: <\%= ENV['OS_SSH_KEYPAIR'] %>
-  private_key_path: <\%= ENV['OS_PRIVATE_SSH_KEY'] %>
-  public_key_path: <\%= ENV['OS_PUBLIC_SSH_KEY'] %>
-  openstack_tenant: <\%= ENV['OS_TENANT_NAME'] %>
-  floating_ip_pool: <\%= ENV['OS_FLOATING_IP_POOL'] %>
-  flavor_ref: <\%= ENV['OS_FLAVOR_REF'] %>
+  openstack_username: <%%= ENV['OS_USERNAME'] %>
+  openstack_api_key: <%%= ENV['OS_PASSWORD'] %>
+  openstack_auth_url: <%%= "#{ENV['OS_AUTH_URL']}/tokens" %>
+  key_name: <%%= ENV['OS_SSH_KEYPAIR'] %>
+  private_key_path: <%%= ENV['OS_PRIVATE_SSH_KEY'] %>
+  public_key_path: <%%= ENV['OS_PUBLIC_SSH_KEY'] %>
+  openstack_tenant: <%%= ENV['OS_TENANT_NAME'] %>
+  floating_ip_pool: <%%= ENV['OS_FLOATING_IP_POOL'] %>
+  flavor_ref: <%%= ENV['OS_FLAVOR_REF'] %>
+  # Only use the following if you need to open up firewall ports
+  security_groups:
```

Comment:
This should be removed from the default template. Please look at the docs to align it with our current standards.